Misc/NEWS.d/next/Library/2025-09-02-10-23-09.gh-issue-116946.U6RpwK.rst

```diff
@@ -0,0 +1,2 @@
+The :class:`!_random.Random` C type is now immutable. Patch by Bénédikt
+Tran.
```

Comment:
I don't think that the change deserves a NEWS entry, since `random.Random` remains mutable, no?

Reply:
Ok, it makes sense.

Reply:
Note that `_random` is an internal module. Very few people use `_random.Random`. What is the reason if most useful methods are in `random.Random`?